models/api.models.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs/promises");
+const endpoints = require("../endpoints");
+
+async function fetchEndpoints() {
+  const data = await fs.readFile("./endpoints.json", "utf8");
+  return JSON.parse(data);
+}
```

Comment:
Nice, two ways to go about it this and just requiring the file endpoints file in and sending it, one isn't necessarily better than the other.
A model really is only for interacting with our database which this isn't doing so I'd move the reading of the file into the controller and a model wouldn't be needed for this task.